docs/roman/source_detection/arguments.rst

```diff
@@ -42,7 +44,7 @@ in a parameter file.
 * ``--bkg_boxsize``: If using `calc_threshold_img` size of box in pixels for
                      2D background / threshold images and if using
                      calc_threshold_2d, the size of the box used when detecting
-                     sources. Default is 3.
+                     sources. Default is 9.
```

Comment:
Good catch!  9 is pretty tiny too.  I think we use 256 for the equivalent of this in other surveys!

Reply:
This number will still need to be adjusted if there's a crowding, or the source is very bright, etc.